site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Description: This recipe describes how to use rclone with MinIO Server.
+        SEO Keywords: Rclone
 
       - Setup Apache HTTP proxy with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/setup-apache-http-proxy-with-minio.md
         Slug: setup-apache-http-proxy-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Setup Apache HTTP proxy with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to set up Apache HTTP with mod_proxy module for connecting to MinIO Server. We are going to set up a new VirtualHost for example.com
+        SEO Keywords: Apache HTTP
 
       - Upload files from browser using pre-signed URLs:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/presigned-put-upload-via-browser.md
         Slug: upload-files-from-browser-using-pre-signed-urls
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Upload files from browser using pre-signed URLs - Cookbook/Recipe
+        SEO Description: This guide describes how to use the presigned PutObject API from the MinIO JavaScript Library to generate a pre-signed URL.
+        SEO Keywords: pre-signed URLs
 
       - How to run Minio in FreeNAS?:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/running-minio-in-freenas.md
         Slug: how-to-run-minio-in-freenas
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to run MinIO in FreeNAS? - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to run MinIO with FreeNAS.
+        SEO Keywords: FreeNAS
 
       - How to use AWS SDK for PHP with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-php-with-minio.md
         Slug: how-to-use-aws-sdk-for-php-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for PHP with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-php is the official AWS SDK for the PHP programming language. In this recipe we will learn how to use aws-sdk-php with MinIO server.
+        SEO Keywords: AWS SDK for PHP
 
       - How to use AWS SDK for Ruby with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-ruby-with-minio.md
         Slug: how-to-use-aws-sdk-for-ruby-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Ruby with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk is the official AWS SDK for the Ruby programming language. In this recipe we will learn how to use aws-sdk for Ruby with MinIO server
+        SEO Keywords: AWS SDK for Ruby, AWS SDK for Ruby on Rails
```

Comment:
We might want to indicate on the keywords that this document shows how to use the AWS SDK for Ruby on the MinIO server